bundles/framework/divmanazer/component/VisualizationForm.js

```diff
@@ -129,6 +129,96 @@ Oskari.clazz.define(
             return values;
         },
 
+        /**
+         * @public @method getOskariStyle
+         * Returns Oskari JSON style
+         *
+         * @return {Object}
+         */
+        getOskariStyle: function () {
+            const values = this.getValues();
+            var oskariStyle = {
+                fill: {
+                    area: {
+                        color: values.area.fillColor,
```

Comment:
In mapmodule getStyle() the fill color is just:

    {
        fill: {
            "color": "#FFBB00"
        },
        ...
    }

https://github.com/oskariorg/oskari-frontend/blob/develop/bundles/mapping/mapmodule/MapModuleClass.ol.js#L861

Reply:
After this small change this looks good for merging